CONTRIBUTING.md

```diff
@@ -44,8 +44,10 @@ F3D uses [GitLab Flow](https://about.gitlab.com/topics/version-control/what-is-g
 - [Fork](https://github.com/f3d-app/f3d/fork) the F3D repository on GitHub.
 - Create and push a feature branch on your fork containing new commits.
 - When it is ready for review or when you want to run the CI, create a pull request against `f3d-app/f3d/master`.
-- Once the PR is approved and CI comes back clean, a F3D maintainer will merge your pull request in the master branch.
-- The master now contains your changes and will be present in the next release!
+- Your PR will be reviewed by maintainers and returning contributors, please take their feedback into account and resolve discussions when adresssed.
+- Maintainers and returning contributors will run the continuous integration and help you figure out how to fix CI issues.
+- Once the PR is approved and CI comes back clean, a maintainer will merge your pull request in the master branch.
+- The master now contains your changes and will be present in the next minor/major release!
```

Comment:
Maybe specify that any documentation improvement is reflected immediately on the website?